packages/cli/src/lib/quire/init-starter.js

```diff
@@ -1,38 +1,56 @@
+import { basename, join, resolve } from 'node:path'
 import { cwd } from 'node:process'
 import fs from 'fs-extra'
 import git from '#src/lib/git/index.js'
-import path from 'node:path'
 /**
  * Clone or copy a Quire starter project
  *
  * @param    {String}   starter   A repository URL or path to local starter
- * @param    {String}   rootPath  Absolute system path to the project root
+ * @param    {String}   projectRoot  Absolute system path to the project root
  * @return   {Promise}
  */
-export async function initStarter (starter, rootPath, quireVersion) {
+export async function initStarter (starter, projectRoot, quire11tyVersion) {
   const remote = 'https://github.com/thegetty/quire-starter-default'
 
   // Clone starter project repository
   await git
-    .cwd(rootPath)
+    .cwd(projectRoot)
     .clone(remote, '.')
     .catch((error) => console.error('[CLI:error] ', error))
 
+  // write project config to project
+  const projectConfig = {
+    projectRoot: resolve(projectRoot),
+    version: quire11tyVersion
+  }
+  const configFilePath = join(projectRoot, 'project.json')
+  const configJSON = JSON.stringify(projectConfig, null, 2)
+  fs.writeFileSync(configFilePath, configJSON)
+
   // Copy 11ty files
-  const fullRootPath = path.resolve(rootPath)
-  const eleventyPath = path.resolve(cwd(), path.join('quire', 'versions', quireVersion))
-  const eleventyFiles = fs.readdirSync(eleventyPath)
+  const fullProjectRootPath = resolve(projectRoot)
+  const eleventyPath = resolve(cwd(), join('quire', 'versions', quire11tyVersion))
+  // @TODO remove `.env` and `_site` from published `quire-11ty` NPM package
+  const eleventyIgnoreFiles = ['.env', '_site']
```

Comment:
@anderspollack `@thegetty/quire-11ty` has been updated to use the `.ignore` file when publishing and unpublished, it can be republished in 24 hours.
```suggestion
```